index.html

```diff
@@ -291,7 +290,7 @@ <h2 id='stipends'>Stipends</h2>
                 with a short letter of support.
                 </p>
 
-                <h2 id='submissions'>Submissions</h2>
+                <h2>Submissions</h2>
```

Comment:
Doesn't harm to keep the id in case people want to link to it – just let's not bloat the menu